Solutions/OCILogs/Data Connectors/azuredeploy_OCI_logs_API_FunctionApp.json

```diff
@@ -0,0 +1,254 @@
+            "name": "[variables('FunctionName')]",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', tolower(variables('FunctionName')))]",
+                "[resourceId('Microsoft.Insights/components', variables('FunctionName'))]"
+            ],
+            "kind": "functionapp,linux",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "name": "[variables('FunctionName')]",
+                "httpsOnly": true,
+                "clientAffinityEnabled": true,
+                "alwaysOn": true,
+                "reserved": true,
+                "siteConfig": {
+                    "linuxFxVersion": "python|3.8"
+                }
+            },
+
+            "resources": [
+                {
+                    "apiVersion": "2018-11-01",
+                    "type": "config",
+                    "name": "appsettings",
+                    "dependsOn": [
+                        "[concat('Microsoft.Web/sites/', variables('FunctionName'))]"
+                    ],
+                    "properties": {
+                        "FUNCTIONS_EXTENSION_VERSION": "~3",
+                        "FUNCTIONS_WORKER_RUNTIME": "python",
+                        "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('Microsoft.insights/components', variables('FunctionName')), '2015-05-01').InstrumentationKey]",
+                        "APPLICATIONINSIGHTS_CONNECTION_STRING": "[reference(resourceId('microsoft.insights/components', variables('FunctionName')), '2015-05-01').ConnectionString]",
+                        "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('FunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('FunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                        "user": "[parameters('user')]",
+                        "key_content": "[parameters('key_content')]",
+                        "pass_phrase": "[parameters('pass_phrase')]",
+                        "fingerprint": "[parameters('fingerprint')]",
+                        "tenancy": "[parameters('tenancy')]",
+                        "region": "[parameters('region')]",
+                        "MessageEndpoint": "[parameters('MessageEndpoint')]",
+                        "StreamOcid": "[parameters('StreamOcid')]",
+                        "AzureSentinelWorkspaceId": "[parameters('AzureSentinelWorkspaceId')]",
+                        "AzureSentinelSharedKey": "[parameters('AzureSentinelSharedKey')]",
+                        "logAnalyticsUri": "[variables('LogAnaltyicsUri')]",
+                        "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/Azure/Azure-Sentinel/raw/master/Solutions/OCILogs/Data%20Connectors/OCILogsConn.zip"
```

Comment:
Here short link should be used in place of zip folder URL.

Reply:
done